src/components/SccRemoteImage.tsx

```
@@ -0,0 +1,92 @@
+import React, {useEffect, useState} from 'react';
+import {Image, ViewStyle, ImageResizeMode, LayoutChangeEvent} from 'react-native';
```

ESLint

[error] Replace `Image,·ViewStyle,·ImageResizeMode,·LayoutChangeEvent` with `⏎··Image,⏎··ViewStyle,⏎··ImageResizeMode,⏎··LayoutChangeEvent,⏎`

(prettier/prettier)